cypress/integration/code-schools.spec.js

```diff
@@ -14,4 +14,31 @@ describe('code schools', function() {
     // 40 is arbitrary, but it proves that the API is working and leading to rendered content
     cy.get('[data-testid="SchoolCard"]').should('have.length.greaterThan', 40);
   });
+
+  it('should render Cincy and Tech Elevator after selecting Ohio', () => {
+    cy.wait('@codeSchools');
+    cy.get('input#react-select-state_select-input')
```

Comment:
Try passing `data-testid` to the react select component. Then you can target a selector that's human-readable.

Reply:
Im going to have to work on this one in a bit. I can pass an id, but will need to do a custom.component for a custom selector. I just saw your latest pull request and will shift fire once i hop on my laptop

Reply:
I don't think you'll need a custom component. You can simply pass `data-testid` to the `ThemedReactSelect` component on the code schools page

Reply:
I will try it again. Last night at 2 am i was getting errors left and right for it, maybe i was just too tired.

Reply:
Kyle, after doing so, data-testid is not showing up as a selector. Am I doing it wrong?
<img width="1287" alt="Screen Shot 2019-05-07 at 11 05 15 AM" src="https://user-images.githubusercontent.com/30167817/57322539-a2da0000-70b8-11e9-8568-cade8ac9b66e.png">

<img width="1287" alt="Screen Shot 2019-05-07 at 11 08 47 AM" src="https://user-images.githubusercontent.com/30167817/57322558-abcad180-70b8-11e9-85e6-bf64db4df0ac.png">

Reply:
No, you're right.

Looks like [that's unfortunately unexpected](https://github.com/JedWatson/react-select/issues/3149).

Go ahead and continue selecting the component as you were initially.

Reply:
Yeah that was the one I was looking at this morning. Ok revert back